kpis/kpi_calculator.py

```diff
@@ -630,6 +666,78 @@ def get_computational_time_ratio(self):
 
         return time_rat
 
+    def get_actuator_travel(self):
+        '''This method returns the measure of the actuator travel displacement when accounting for the sum of all
+        the actuators present in the test case.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        ltra_tot: float
+            displacement of actuator travel
+
+        '''
+        self.ltra_tot = 0.
+        # Calculate displacement of actuator travel
+
+        for source in self.sources_ltra:
+            for signal in self.ltra_source_key_mapping[source]:
+                ltra_data = np.array(self._get_data_from_last_index(signal,self.i_last_ltra))
+                index_data = np.array(self._get_data_from_last_index('time',self.i_last_ltra))
+
+                # Calculate displacement and update dictionaries
+                displacement = self._displacement(index_data, ltra_data, index_data[0], index_data[-1])
+                self.ltra_dict[signal] += displacement
+                self.ltra_dict_by_source[source + '_' + signal] += self.ltra_dict[signal]
+                self.ltra_tot += self.ltra_dict[signal]
+
+        # Assign to case
+        self.case.ltra_tot            = self.ltra_tot/len(self.ltra_dict) #Divide by the number of the control actuators
```

Comment:
Suggest following the same pattern as other KPIs for consistency in normalization.  See line 584 for energy cost, where the division by floor area happens when calculating the total on the fly.  I suggest similar here: move the ``/len(self.ltra_dict)`` to line 695.